Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -161,7 +160,15 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         overlayView.addSubview(logoImageView)
         logoImageView.snp_makeConstraints({ (make) -> Void in
             make.centerX.equalTo(overlayView)
-            make.top.equalTo(overlayView.snp_top).offset(20)
+
+            // Sets proper constraint for iPads.
+            make.centerY.lessThanOrEqualTo(overlayView.snp_centerY).priorityHigh()
+
+            // Sets proper top constraint for iPhone 6 in portait.
+            make.centerY.equalTo(overlayView.snp_centerY).offset(-160).priorityMedium()
+
+            // Sets proper top constraint for iPhone 4, 5 in portrait.
+            make.top.greaterThanOrEqualTo(overlayView.snp_top).offset(50).priorityHigh()
```

Comment:
Hm, these priorities are a bit confusing, and I'm not sure how they work. Could you add some comments describing why the priority levels are set the way they are? But as long as this works, I guess it's fine to me.